docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -90,6 +90,7 @@ Alpha features might be changed or removed without prior notice.
 | `logsContextDatasourceUi`          | Allow datasource to provide custom UI for context view                                                                                                                       |
 | `lokiQuerySplitting`               | Split large interval queries into subqueries with smaller time intervals                                                                                                     |
 | `individualCookiePreferences`      | Support overriding cookie preferences per user                                                                                                                               |
+| `onlyExternalOrgRoleSync`          | Enables organization roles synced with external auth providers to not be changed at all                                                                                      |
```

Comment:
```suggestion
| `onlyExternalOrgRoleSync`          |  Prohibits a user from changing organization roles synced with external auth providers                                                                                      |
```

Reply:
this file is generated automatically, so i will change it in the registry of the featuretoggle